odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -1209,13 +1213,13 @@ PROCEDURE ERRORHANDLER
 *]
  ERRORMSG="loggingErrorMessage"
  ERRORDETAIL(0)="Program running in memo mode"
- ERRORDETAIL(1)="Error reading from config file:"+LFERROR
- ERRORDETAIL(2)="Config file validation error"
+ ERRORDETAIL(1)="Error reading from config file: "+READCONFIGERRORMSG
```

Comment:
Hi Tom, I see your Point about the Read config error in the ODT Opt-In PO changes for the Error Codes.
Do you prefer I change the code back so it ends up as "Error reading from config file: [LFERROR]" (Just adding the missing space after the :) 

Or would you prefer I change it so the two error messages end up like the following?:
Config file error: Error Opening Letterfile [configuration file name]: [system generated letter file read error message]
and
Config file error: Error Reading Letterfile [configuration file name]: [system generated letter file read error message]

Reply:
@jordan-keenan I really don't have a preference honestly, It just seemed like you'd be stacking two error messages on top of each other, and it can be confusing. 

Maybe, if you would rather use the 'READCONFIGERRORMSG' value then drop the 'Error reading from config file: ' portion of the error message and just make it 'ERRORDETAIL(1)=READCONFIGERRORMSG'

Would that work?

Reply:
@tkainz. Thanks, I've made your suggested change and resubmitted the code review.

Reply:
@jordan-keenan After the mod, did you test it out to see/verify the change will work as intended?